bitcoin/tests/serde.rs

```diff
@@ -296,7 +296,7 @@ fn serde_regression_psbt() {
                 "304402204f67e2afb76142d44fae58a2495d33a3419daa26cd0db8d04f3452b63289ac0f022010762a9fb67e94cc5cad9026f6dc99ff7f070f4278d30fbc7d0c869dd38c7fe701".parse().unwrap(),
             )].into_iter().collect(),
             bip32_derivation: keypaths.clone().into_iter().collect(),
-            final_script_witness: Some(Witness::from_vec(vec![vec![1, 3], vec![5]])),
+            final_script_witness: Some(Witness::from_slice(&[vec![1, 3], vec![5]])),
```

Comment:
Nit: the inner vecs could've been changed to slices but let's keep it for some bored students. :)